model/task/sorter.go

```diff
@@ -1,12 +1,8 @@
 package task
 
-import (
-	"github.com/evergreen-ci/evergreen/db"
-	"github.com/mongodb/grip"
-	"github.com/mongodb/grip/message"
-)
+import "github.com/evergreen-ci/evergreen/db"
 
-type Tasks []Task
+type Tasks []*Task
```

Comment:
why do we need this? are we setting ourselves up for a range-over-slice-of-pointers bug?

Reply:
I would unwind if possible.

Reply:
This was the definition before - my changes in this file are a revert